frontend/apps/reader/modules/readerview.lua

```diff
@@ -644,7 +679,7 @@ function ReaderView:drawHighlightRect(bb, _x, _y, rect, drawer, draw_note_mark)
                 note_mark_pos_x = self.note_mark_pos_x2
             end
             if self.highlight.note_mark == "sideline" then
-                bb:paintRect(note_mark_pos_x, y, self.note_mark_line_w, h, Blitbuffer.COLOR_BLACK)
+                bb:paintRect(note_mark_pos_x, y, self.note_mark_line_w, h, color)
```

Comment:
Should fallback to black.

Reply:
It does fall back to black. See line 668.